src/test/java/cucumber/steps/Hooks.java

```diff
@@ -0,0 +1,33 @@
+package cucumber.steps;
+
+import api.ApiClient;
+import exeptions.FailedToLoginException;
+import helpers.UserTokenGenerator;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.BeforeAll;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Hooks {
+    private static final Logger LOGGER = LogManager.getLogger();
+    public static ApiClient api = new ApiClient();
+    private static UserTokenGenerator userToken = new UserTokenGenerator(api);
+
+    @BeforeAll
+    public static void setAccessToken() throws FailedToLoginException {
+        LOGGER.info("Setting token ...");
+        userToken.setToken();
```

Comment:
not appropriate name for generator

Reply:
changed classname to UserLogin and method to login()